conf/jira.yaml.template

```diff
@@ -0,0 +1,5 @@
+JIRA:
```

Comment:
Just a thought here, Can we merge both bugzilla.yaml and jira.yaml to one config file?

Reply:
We plan to remove the code for bugzilla once we stop using it so I'd rather keep things separate for simplicity.